src/Components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <>
+    <div>
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+    <div>
+      <Button name="7" />
+      <Button name="9" />
+      <Button name="8" />
+      <Button name="X" />
+    </div>
+    <div>
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
+      <Button name="-" />
+    </div>
+    <div>
+      <Button name="1" />
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div>
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
+    </div>
```

Comment:
[REQUIRED] These buttons should be arranged in the following groups:
![image](https://user-images.githubusercontent.com/73254040/97767639-f9c06a00-1ae2-11eb-8250-12ac02dfeed2.png)